### ISSUE TITLE
Cut release 11

### ISSUE THREAD

**Issue by pchickey** [WebAssembly/wasi-sdk#139]
I'd like to cut release 11 this week.

Since release 10, these changes are merged in wasi-libc:
https://github.com/WebAssembly/wasi-libc/pull/197
https://github.com/WebAssembly/wasi-libc/pull/198

These outstanding PRs to wasi-libc seem reasonable to get in as part of this release:
https://github.com/WebAssembly/wasi-libc/pull/173
https://github.com/WebAssembly/wasi-libc/pull/199

Since release 10, these changes are merged in wasi-sdk:
https://github.com/WebAssembly/wasi-sdk/pull/123
https://github.com/WebAssembly/wasi-sdk/pull/124
https://github.com/WebAssembly/wasi-sdk/pull/126
https://github.com/WebAssembly/wasi-sdk/pull/130
https://github.com/WebAssembly/wasi-sdk/pull/132
https://github.com/WebAssembly/wasi-sdk/pull/136

These outstanding PRs to wasi-sdk seem reasonable to get in as part of this release:
https://github.com/WebAssembly/wasi-sdk/pull/134
https://github.com/WebAssembly/wasi-sdk/pull/137

cc @whitequark

**Comment by sbc100** [comment]
I don't think we need to block on #134, there are some parts of that the might land, others not.

**Comment by pchickey** [comment]
Tag pushed, the release artifacts are being built by the CI run here: https://github.com/WebAssembly/wasi-sdk/actions/runs/124193016

**Comment by whitequark** [comment]
Looks like the build is done!

**Comment by pchickey** [comment]
And artifacts and notes are uploaded. Thanks everyone! https://github.com/WebAssembly/wasi-sdk/releases/tag/wasi-sdk-11
